### ISSUE TITLE
Newline inserted at end of <script> or <style> when `end_with_newline` enabled

### ISSUE THREAD

**Issue by jawshooah** [beautify-web/js-beautify#644]
js-beautify always inserts a newline at the end of a `<script>` or `<style>` block, and I can't find an option to turn this behavior off.

Before:

``` html
<style type="text/css">
    body {
        display: none;
    }
</style>

<script type="text/javascript">
    (function() {
        console.log("Hello, World");
    })();
</script>
```

After:

``` html
<style type="text/css">
    body {
        display: none;
    }

</style>

<script type="text/javascript">
    (function() {
        console.log("Hello, World");
    })();

</script>
```

If there is a newline at the start of the tag, it deletes it as well:

Before:

``` html
<style type="text/css">

    body {
        display: none;
    }
</style>

<script type="text/javascript">

    (function() {
        console.log("Hello, World");
    })();
</script>
```

After:

``` html
<style type="text/css">
    body {
        display: none;
    }

</style>

<script type="text/javascript">
    (function() {
        console.log("Hello, World");
    })();

</script>
```

**Comment by bitwiseman** [comment]
The option that you want is `end_with_newline=false`.  Where are you running this from?

**Comment by jawshooah** [comment]
I'm running `grunt-jsbeautifier` with `js-beautify` version 1.5.5.

I'd rather not disable `end_with_newline`, as I still want the file itself to end with a newline. It's just that the newline at the end of the script/style block looks weird and out of place to me.

It might be a different if the newline at the start of the script/style block were allowed to remain, as that would provide clear, consistent visual separation of the JS/CSS from the HTML.

**Comment by bitwiseman** [comment]
Indeed.  See #391, which a actually references back into the `grunt-jsbeautifier`.  Separating these options, maybe making them default to the root value, probably wouldn't be too hard, but not the highest priority.

**Comment by eliashdezr** [comment]
Any news on this? I'm facing the same issue. This is a little annoying when working with Vue, where you have the `script` and `style` tags on the same file.

**Comment by MacKLess** [comment]
This has been addressed and resolved (with tests) when I checked using v1.8.0-rc12. See the Readme (https://github.com/beautify-web/js-beautify#setting-inheritance-and-language-specific-overrides) for more information. This is also the case for Issue #639.
